res/parking/parking_manager.c

```diff
@@ -93,6 +93,9 @@
 			<parameter name="Parkinglot" required="false">
 				<para>The parking lot to use when parking the channel</para>
 			</parameter>
+			<parameter name="Parkingpos" required="false">
+				<para>The parking space extension in the parking lot</para>
```

Comment:
State what happens if the position is already taken

Reply:
And make it `ParkingPosition` instead of `Parkingpos`.

Reply:
Using ParkingSpace